user/enterprise/setting-up-travis-ci-enterprise.md

```diff
@@ -0,0 +1,144 @@
+    | 22   | SSH             | SSH access.                                                                  |
+
+1. If you're using a hostname and not just an IP address, this would be a time to configure it.
+
+1. *On your new virtual machine*, download and run the installation script:
+
+
+    ```
+    curl -sSL -o /tmp/installer.sh https://enterprise.travis-ci.com/install
+    sudo bash /tmp/installer.sh
+    ```
+
+3. *In your browser*, navigate to `https://<hostname>:8800` (your Enterprise
+installation's hostname, port 8800) to complete the setup:
+
+   1. Add a secure certificate or configure a trusted one.
+   1. Upload your Travis CI Enterprise license.
+   1. Configure access to the Admin Console with a password or using openLDAP. This controls access to the Admin Console itself, not to the Travis CI Enterprise instance.
+   1. Connect your GitHub Enterprise or GitHub.com with Travis CI enterprise.
+   1. Optionally, configure Email, Metrics and Caches.
+   1. Copy the *RabbitMQ password* for the Worker setup.
+
+
+## 2. Setting up the Enterprise Worker virtual machine
+
+The Travis CI Enterprise Worker manages build containers and reports build
+statuses back to the platform. It must be installed on a separate machine
+instance from the Platform. We recommend using AWS' `c4.2xlarge` instance running Ubuntu 16.04 LTS or later as the underlying operating system.
+
+Make sure you have already [set up the Enterprise Platform](/user/enterprise/setting-up-travis-ci-enterprise/#1-setting-up-enterprise-platform-virtual-machine) and have the *RabbitMQ password* and the *hostname* from the Platform Dashboard.
+
+
+1. *On your virtual machine management platform*, create a Travis CI Worker Security Group
+
+    If you're setting up your AMI for the first time you will need to create
+    a Security Group. From the EC2 management console, create an entry for
+    each port in the table below:
+
+    | Port | Service | Description |
+    |:-----|:--------|:------------|
+    | 22   | SSH     | SSH access. |
+
+1. *On your new virtual machine*, download and run the installation script:
+
+    ```
+    $ curl -sSL -o /tmp/installer.sh https://raw.githubusercontent.com/travis-ci/travis-enterprise-worker-installers/master/installer.sh
+    $ sudo bash /tmp/installer.sh --travis_enterprise_host="<enterprise host>" --travis_enterprise_security_token="<rabbitmq password>"
+    ```
+
+### Installing workers behind a web proxy
+
+If you are behind a web proxy and Docker fails to download the image(s), when you run the worker installation script, edit `/etc/default/docker` and set your proxy there.
+Then rerun the installation script.  
+
+If you need Docker itself to use an HTTP proxy, export it before each docker command:
+
+```
+export http_proxy="http://proxy.mycompany.corp:8080/" docker <COMMAND>
+```
+
+### Older versions of Travis CI Enterprise
```

Comment:
I'm not 100% sure what the reference of older versions helps with here. Do we want new setups on any of these? @danishkhan